src/main/java/seedu/dictionote/logic/commands/DeleteNoteCommand.java

```diff
@@ -0,0 +1,53 @@
+package seedu.dictionote.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.dictionote.commons.core.Messages;
+import seedu.dictionote.commons.core.index.Index;
+import seedu.dictionote.logic.commands.exceptions.CommandException;
+import seedu.dictionote.model.Model;
+import seedu.dictionote.model.note.Note;
+
+/**
+ * Deletes a note identified using it's displayed index from the contacts list.
```

Comment:
Shouldn't it be `from the notes list` instead?
This issue appears in other parts of the code as well